torchmetrics/text/ter.py

```diff
@@ -0,0 +1,128 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, Callable, List, Optional, Sequence, Tuple, Union
+
+import torch
+from torch import Tensor, tensor
+
+from torchmetrics.functional.text.ter import _ter_compute, _ter_update, _TercomTokenizer
+from torchmetrics.metric import Metric
+
+
+class TER(Metric):
+    """Calculate Translation edit rate (`TER`_)  of machine translated text with one or more references. This
+    implementation follows the implmenetaions from
+    https://github.com/mjpost/sacrebleu/blob/master/sacrebleu/metrics/ter.py. The `sacrebleu` implmenetation is a
```

Comment:
should it be `sacreblue` and not `ter` here?

Reply:
Actually, I refer here to the `sacrebleu` (package name) implementation of `TER`. So I'm not sure how to properly formulate that.